pkgs/applications/misc/octoprint/default.nix

```diff
@@ -249,6 +255,8 @@ let
                 "test_cookies.py"
                 # requires network
                 "test_worker.py"
+                # fails on aarch64
+                "test_server_events.py"
```

Comment:
Is this safe to ignore, or should we rather mark the package as broken on aarch64?

Reply:
Octoprint is specifically for aarch64 (raspberry pi).
It makes no sense to mark it as broken on aarch64.

Most of those quirky version and test overrides can be removed, once octoprint moves to the next version (see pr #165328)

Reply:
According to the website it looks like this could just as well be run on x86. My question is basically the same as in the other thread: Does this test failure indicate any issues that a user might see during actual usage?

If a new version that would fix these issues is already in release-candidate stage, maybe it would be better to use that instead. @gebner what do you think?

Reply:
Well, the new version would be awesome. It removes most of the version entanglement. 
It works well for me :tm: 

The issue is, that it doesn't build, if we keep the tests.

Reply:
Great! Then I'll close this for now. I want to give the maintainers of octoprint (@abbradar, @gebner, @WhittlesJr) some time to review the update to the release candidate (#165328). Please remind me some time after Monday if nobody has responded.

Reply:
@timokau could you have a look at the release candidate pr? Unfortunately, I doesn't build because of an upstream build error of sentry-sdk (#169130), but worked fine before, and should build fine again once the issues in sentry-sdk are resolved.

Reply:
Done, sorry for the delay.